release.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
             '@semantic-release/exec',
             {
                 prepareCmd:
-                    'npm version ${nextRelease.version} --workspaces --include-workspace-root --allow-same-version --no-git-tag-version 2>&1 | grep -q EUNSUPPORTEDPROTOCOL && exit 0 || exit 1',
+                    'npm version ${nextRelease.version} --workspaces --include-workspace-root --allow-same-version --no-git-tag-version 2>&1 | grep -q -E "EUNSUPPORTEDPROTOCOL|Invalid comparator',
```

Comment:
this is the only change compared to the prev branch

Reply:
`Invalid comparator` happens because of `npm:` prefixed package refs:
```
        "@mantine-8/core": "npm:@mantine/core@8.0.0",
        "@mantine-8/hooks": "npm:@mantine/hooks@8.0.0",
```